web/client/epics/catalog.js

```diff
@@ -265,7 +266,9 @@ export default (API) => ({
                 const actions = [];
                 const state = store.getState();
                 const id = getLayerId(layer);
-                actions.push(addNewLayer({...layer, id}));
+                if (layer.type !== 'arcgis') {
+                    actions.push(addNewLayer({...layer, id}));
+                }
```

Comment:
finally we can restore this because it's not used by arcgis type anymore
```suggestion
                actions.push(addNewLayer({...layer, id}));
```

Reply:
Removed here https://github.com/geosolutions-it/MapStore2/pull/10293/commits/fc115a23255e03cef0cda6992ab3ff195b6b3d03